astropy/io/fits/convenience.py

```diff
@@ -725,7 +724,7 @@ def tabledump(filename, datafile=None, cdfile=None, hfile=None, ext=1,
     -----
     The primary use for the `tabledump` function is to allow editing in a
     standard text editor of the table data and parameters.  The
-    ``tcreate`` function can be used to reassemble the table from the
+    ``tableload`` function can be used to reassemble the table from the
```

Comment:
No reason I can think of, unless it's one of those odd cases where the reference isn't resolved for some reason.